test/patterns.js

```diff
@@ -38,3 +37,10 @@ definePattern('includedIn', (target, set) =>
 definePattern('trueOrUndefined', value =>
   value === true || value === undefined
 );
+
+definePattern('anAssetUrl', /https?:\/\/.+/);
+
+definePattern('aUid', /^[a-z0-9]{32}$/);
+
+require ('./publicProfile.matchers');
+require ('./organizationMemberships.matchers');
```

Comment:
no

Reply:
we are not putting every single matcher in one giant file.

Reply:
got it, removed